homeassistant/components/circuit/notify.py

```diff
@@ -0,0 +1,45 @@
+"""Unify Circuit platform for notify component."""
+import logging
+
+import attr
+from circuit_webhook import Circuit
+
+from homeassistant.components.notify import ATTR_TARGET, BaseNotificationService
+from homeassistant.const import CONF_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Unify Circuit notification service."""
+    if discovery_info is None:
+        return None
+
+    try:
+        return CircuitNotificationService(hass, discovery_info)
+
+    except RuntimeError as err:
```

Comment:
Why would there be an exception here? It doesn't look possible.

Reply:
Okay, make sense will change it.